src/clj/game/cards-resources.clj

```diff
@@ -137,6 +137,25 @@
    {:in-play [:hand-size-modification 5]
     :events {:runner-turn-begins {:msg "lose [Click]" :effect (effect (lose :click 1))}}}
 
+   "Beth Kilrain-Chang"
+   (let [ability {:once :per-turn
+                  :label "Gain 1 [Credits], draw 1 card, or gain [Click] (start of turn)"
+                  :req (req (:runner-phase-12 @state))
+                  :effect (req (let [c (:credit corp)
+                                 b (:title card)]
+                             (when (<= 5 c 9)
```

Comment:
Probably? I looked at the use of `cond` elsewhere and for some reason got scared off and didn't try it.